qemu/tests/qemu_img_check_data_integrity.py

```diff
@@ -89,3 +92,17 @@ def run_iozone_background(vm):
         session = vm.wait_for_login()
         session.cmd(params["rm_testfile_cmd"] % iozone_testfile)
     session.close()
+    vm.destroy()
+
+    # Check the testing image and repair the leaked clusters
+    root_dir = data_dir.get_data_dir()
+    img_param = params.object_params("image1")
+    img = QemuImg(img_param, root_dir, "image1")
+    res = img.check(params, root_dir)
+    if res.exit_status == 2:
+        test.fail("Image is corrupted, check please.")
+    elif res.exit_status == 3:
+        test.log.info("Leaked clusters found in the image, repair them now.")
+        res = img.check(params, root_dir, check_repair="leaks")
+        if res.exit_status != 0:
+            test.fail("Repair the leaked image failed, check please.")
```

Comment:
do we need additional check besides the image.check? e.g. 
    start vm again and try to log into it, there should no error
That can make sure repairing the image doesn't lead to a corruption.

Reply:
same question as zhenchao, do we need to start vm again?

Reply:
Updated, thanks.